e2e/vm/lifecycle_test.go

```diff
@@ -23,7 +23,14 @@ var testVMLifecycle = func(o *option.Option) {
 				gomega.Expect(command.StdoutStr(o, virtualMachineRootCmd, "status")).To(gomega.Equal("Running"))
 			})
 
+			ginkgo.It("should be able to force remove the virtual machine", func() {
+				command.Run(o, "images")
+				command.New(o, virtualMachineRootCmd, "remove", "--force").WithTimeoutInSeconds(60).Run()
+				command.RunWithoutSuccessfulExit(o, "images")
+			})
+
 			ginkgo.It("should be able to stop the virtual machine", func() {
+				command.New(o, virtualMachineRootCmd, "init").WithTimeoutInSeconds(600).Run()
```

Comment:
nit: maybe we can move this line to the last `It` so that for every `It` inside this `When` block (i.e., `"the virtual machine is in running status"`), the VM is running.